src/prommis/uky/costing/ree_plant_capcost.py

```diff
@@ -1731,6 +1899,23 @@ def get_fixed_OM_costs(
                 f"{list(default_sale_prices.keys())}"
             )
 
+        # set default values
+        if labor_types is None:
```

Comment:
The arguments are set using `None` as a guard value so that a new object is created every time the method is called and there is no chance of the object changing on each run.

Reply:
Sounds good, thank you for clarifying Brandon!